### DISCUSSION TITLE
Is the playground broken?

### DISCUSSION THREAD

**Discussion by lukaslihotzki** [eslint-stylistic/eslint-stylistic#159] [General]
When creating a bug report, the [playground](https://stackblitz.com/fork/github/eslint-stylistic/eslint-stylistic?file=playground%2Feslint.config.js) should be used to create a minimal reproduction. I can edit the files there, but not actually run it, because the "Installing dependencies" phase hangs with 100% CPU for at least 10 minutes. The terminal output until then is:
```
packages/eslint-plugin dev: bundles ./dts/index.ts, ./dts/define-config-support.d.ts, ./dts/rule-options.d.ts → dist...
packages/eslint-plugin-ts dev: created dist in 6.6s
packages/eslint-plugin-jsx dev: created dist in 7.7s
packages/eslint-plugin-js dev: created dist in 11.4s
packages/eslint-plugin dev: created dist in 8.4s
```

Can anyone reproduce this?

**Comment by kecrily** [comment]
This is as expected. You can start a new terminal and run `pnpm run play:lint` to see the results of the lint run.

I'll submit a PR to clarify this.
